hs_app_netCDF/serializers.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import serializers
+
+from hs_core.views.resource_metadata_rest_api import CoreMetaDataSerializer
+from models import OriginalCoverage, Variable, NetcdfMetaData
+
+
+class OrginalCoverageSerializer(serializers.Serializer):
+    value = serializers.SerializerMethodField(required=False)
+    projection_string_type = serializers.CharField(required=False)
+    projection_string_text = serializers.CharField(required=False)
+    datum = serializers.CharField(required=False)
+
+    class Meta:
+        model = OriginalCoverage
+
+    def get_value(self, obj):
+        return obj.value
```

Comment:
@hyi Here is the documentation for that: http://www.django-rest-framework.org/api-guide/fields/#serializermethodfield